channels/OFF-CHAIN.md

```diff
@@ -853,7 +853,7 @@ useable.
 Message code: 18
 
 Since both sides may initiate a `withdraw_created` request, it is possible
-that both may do so at roughly the same time. In particular, in the Aeternity node
```

Comment:
I think there was a discussion about capitalisation, I'm definitely not an English expert, but did you double checked it should be lowercase ?

Reply:
to be discussed. maybe this needs to be addressed in another PR. we should be consistent here

Reply:
any update on this ? consistency can be accompanied by `aeternity` -> `Aeternity` as well.

Reply:
seems like I missed that one. will check again. thanks